doc/cdi-operator-olm.md

```diff
@@ -92,6 +137,23 @@ cluster/kubectl.sh apply -f  _out/manifests/release/cdi-cr.yaml
 ```
 Now the operator should finish its deployment successfully
 
+<a name="update"></a>
+### CDI OLM update
+OLM mechanism supports operator update via subscription mechanism. Once subscription manifest is installed on cluster, it monitors the operator source and when new OLM bundle appears, OLM can trigger update of the operator.
+
+*Note:* Currently CDI operator does **not** support upgrade, but it can be updated via OLM. In such case OLM update will effectivley terminate current _cdi-operaor_ instance and install the new one - specified in the new CSV bundle.
```

Comment:
this sounds a bit confusing to me.... do you mean "does **not** support upgrades of the CDI installation"?

Reply:
Yes, I mean that OLM will update the CDI operator and CDI installation but no upgrade logic will be applied - that is - the state will not be upgraded